### PR TITLE
Update: Kong Gateway overview diagram

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Updating the Kong Gateway overview diagram to 1) remove Dev Portal and Vitals, as both are deprecated, and 2) update the styling to match Konnect and Mesh.

https://konghq.atlassian.net/browse/DOCU-3679

### Testing instructions

Preview link: 

https://deploy-preview-7037--kongdocs.netlify.app/gateway/latest/#packages-and-modes

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

**Comment by lena-larionova** [comment]
❗ Reviewers: I put two images in, so please don't merge yet. Let me know which one makes more sense to you: the hybrid pseudo-architecture image, or the one with just features - or neither, and what would be better.

File: app/_src/gateway/index.md [hunk 1]
```diff
@@ -112,6 +112,13 @@ to only pay for the services your organization uses.
 
 For more information, visit the [pricing page](https://konghq.com/pricing).
 
+![Introduction to {{site.base_gateway}} in {{site.konnect_short_name}}](/assets/images/products/konnect/gateway-manager/konnect-control-planes-example.png)
+> _Figure 1: Diagram of {{site.base_gateway}} data planes connected to a {{site.konnect_short_name}} control plane._
+> <br>
+> _Requests flow from an API client into the
+Gateway data planes, are modified and managed by the proxy based on your control plane
+configuration, and forwarded to upstream services._
```
Comment (cloudjumpercat) [inline app/_src/gateway/index.md:120]:
```suggestion
configuration, and are forwarded to upstream services._
```

**Review by cloudjumpercat — APPROVED** [review]
I left one small grammar comment. These diagrams look fantastic!

**Comment by lena-larionova** [comment]
@cloudjumpercat which diagram do you think works better/makes more sense? I added two Gateway diagrams to compare; we should only keep one.

File: app/assets/images/products/gateway/kong-gateway-features.png
Comment (cloudjumpercat) [inline app/assets/images/products/gateway/kong-gateway-features.png:1]: @lena-larionova I think I like this one better, I think the client/route/service/etc. could be confusing when combined with features. If you want, you can use this diagram for features and you could borrow the data plane node diagram I made here: https://docs.konghq.com/konnect/getting-started/#kong-gateway-entities-in-konnect

**Pull request merged**